x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/remote_cluster_table/remote_cluster_table.js

```diff
@@ -28,42 +28,46 @@ export class RemoteClusterTableUi extends Component {
   constructor(props) {
     super(props);
 
-    const { clusters } = props;
-
     this.state = {
+      queryText: undefined,
       selectedItems: [],
-      searchedClusters: clusters.slice(0),
     };
   }
 
-  onSearch = ({ query }) => {
-    const { clusters } = this.props;
-    const { text } = query;
+  onSearch = (queryObject) => {
+    const { text } = queryObject.query;
```

Comment:
I like this refactor of `onSearch()`. One small thing: why not maintain de deconstruct of `{ query }` ?

Reply:
development leftover, will fix!